packages/ember-glimmer/lib/helpers/-normalize-class.js

```diff
@@ -2,17 +2,21 @@ import { InternalHelperReference } from '../utils/references';
 import { dasherize } from 'ember-runtime/system/string';
 
 function normalizeClass({ positional, named }) {
+  let inactiveClass;
+  let activeClass = positional.at(0).value();
   let value = positional.at(1).value();
-  let activeClass = named.at('activeClass').value();
-  let inactiveClass = named.at('inactiveClass').value();
+
+  if (positional.at(2)) {
+    inactiveClass = positional.at(2).value();
+  }
 
   // When using the colon syntax, evaluate the truthiness or falsiness
   // of the value to determine which className to return.
   if (activeClass || inactiveClass) {
     if (!!value) {
-      return activeClass;
+      return dasherize(activeClass);
     } else {
-      return inactiveClass;
+      return dasherize(inactiveClass);
     }
```

Comment:
I don't think we need to dasherize here – we only do that when you use the "no colon" syntax and the value evaluates to a Boolean. Here (when using the colon syntax)  we expect these to be string literals